andela/travel_tool_front
# #160813365 Remember personal details

#### What does this PR do?
This PR ensures that the personal details of a user are stored and returned in form inputs on next request initiation.


#### Description of Task to be completed?
Currently, a user has to reload the page to have the personal details remembered. however, next time you create a new request the personal details should be remembered.

#### How should this be manually tested?
* Clone this repo branch  - `git clone -b bg-checkbox-state-cached-160371607 https://github.com/andela/travel_tool_front.git`
* Switch to the repo directory - cd travel_tool_front
* Install dependencies and start the server - make start
* Set up the server by following this guide
* Navigate to localhost:5000 in a browser
* sign in using your andela email address
* Click the `New request` button
* Add your personal details, check the checkbox and submit form
* Navigate back to the `New request` button
* your personal details should be in the form inputs

#### Any background context you want to provide?


#### What are the relevant pivotal tracker stories?
[160813365](https://www.pivotaltracker.com/story/show/160813365)

#### Screenshots (if appropriate)
#### unchecked
![image](https://user-images.githubusercontent.com/30747298/46219496-304db880-c350-11e8-9b5e-e9032dc2dd53.png)

##### checked
![image](https://user-images.githubusercontent.com/30747298/46219531-4491b580-c350-11e8-9e96-3bb2e36b946a.png)


#### Questions:
